woocommerce/woocommerce-gateway-stripe
# Fix save payment method checkbox alignment in the CC form when upe is disabled

# Changes proposed in this Pull Request:

fix #2000 

This PR makes the tag hierarchy and positioning of the "Save Payment Method" button match when UPE is enabled or disabled. Previously, the button was shown inside a fieldset when UPE was enabled; and outside of the fieldset when it was disabled.

# Testing instructions

## Test with UPE disabled

1. Activate the Twenty Twenty-One theme
2. Disable UPE
3. Add a product to the cart, and go to the checkout page.
4. The checkbox to save the payment method should be aligned with the credit card fields.

<img width="1063" alt="Screen Shot 2021-10-08 at 15 26 37" src="https://user-images.githubusercontent.com/4153905/136609344-8689e6b3-251a-4cde-b880-973377742423.png">

## Test with UPE enabled

1. Activate the Twenty Twenty-One theme
2. Enable UPE
3. Add a product to the cart, and go to the checkout page.
4. The checkbox to save the payment method should be aligned with the credit card fields.

<img width="1063" alt="Screen Shot 2021-10-08 at 15 27 52" src="https://user-images.githubusercontent.com/4153905/136609425-1548e6eb-16e0-4ba1-8b1e-743eb8511229.png">
